ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<p>Remove this ARN and replace with <code>&lt;&lt;ADD WORK TEAM ARN HERE&gt;&gt;</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Ack</p>